src/spatialdata/dataloader/datasets.py

```diff
@@ -81,6 +81,10 @@ class ImageTilesDataset(Dataset):
         system; this back-transforms the target tile into the pixel coordinates. If the back-transformed tile is not
         aligned with the pixel grid, the returned tile will correspond to the bounding box of the back-transformed tile
         (so that the returned tile is axis-aligned to the pixel grid).
+    return_genes:
```

Comment:
Nice! Two comments: 
1. I would specify that the layers are `AnnData` layers and the default layer is `X`. 
2. I would also allow to pass just a list instead of a dict, that would be interpreted as `{'X': genes_list}`